rails-task-list/app/assets/stylesheets/people.scss

```diff
@@ -0,0 +1,42 @@
+// Place all the styles related to the people controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+a {
+  color:black
+}
+
+
+.person_list {
+  list-style: none;
+  margin-top: 20px;
+}
+
+.person_name_link {
```

Comment:
CSS naming conventions state that we separate words with `-` rather than the `_`